pyzo/core/editor.py

```diff
@@ -259,6 +259,7 @@ def __init__(self, parent, **kwds):
         self.setWrap(bool(pyzo.config.view.wrap))
         self.setHighlightCurrentLine(pyzo.config.view.highlightCurrentLine)
         self.setLongLineIndicatorPosition(pyzo.config.view.edgeColumn)
+        self.setHighlightMatchingBracket(pyzo.config.view.highlightMatchingBracket)
```

Comment:
Does this not crash Pyzo? I'd think that you'd also need to add a default for this in 'resources/defaultConfig.ssdf'